UICatalog/Scenarios/ASCIICustomButton.cs

```diff
@@ -16,15 +17,18 @@ public override void Init ()
 	{
 		Application.Init ();
 		_scrollViewTestWindow = new ScrollViewTestWindow ();
-		var menu = new MenuBar (new [] {
-			new MenuBarItem ("Window Size", new [] {
-				_miSmallerWindow = new MenuItem ("Smaller Window", "", ChangeWindowSize) {
-					CheckType = MenuItemCheckStyle.Checked
-				},
-				null,
-				new MenuItem ("Quit", "", () => Application.RequestStop (), null, null, (KeyCode)Application.QuitKey)
-			})
-		});
+		var menu = new MenuBar {
+			Menus = [
+				new MenuBarItem ("Window Size", new [] {
```

Comment:
I know it's not necessarily in scope for this PR, but I just wanted to point out that anywhere there's a `new[]` or a `new SomeCollectionType();` is a potential and quite likely place that a collection expression is allowed and potentially better, too! 😃 

Others were changed here too, so it seemed relevant to point out. 🤷‍♂️